packages/shared-state/files/usr/lib/lua/shared-state.lua

```diff
@@ -324,11 +326,18 @@ function SharedStateMultiWriter:insert(data)
 	self:notifyHooks()
 end
 
+function shared_state._getFortune()
+	return math.random(1, 100000)
+end
+
 function SharedStateMultiWriter:_insert(key, data)
 	local lv = self.storage[key]
 	if (lv == nil or not utils.deepcompare(lv.data, data)) then
+		local changes = lv and lv.changes + 1 or 0
 		self.storage[key] = {
 			lastModified=os.time(),
```

Comment:
Are we keeping lastModified for some reason? debugging, it could be misleading though...

Reply:
My idea is that having the last modf time may be useful in the future for example for housekeeping.  mean, to have the date already in the db now will allow to know later if we can prune a value or not. But yes it is not useful right know so I am also not very convinced that having it is a good idea.

Reply:
Yeah, I guess it may be helpfull in the future and it doesn't harm no